class.jetpack-options.php

```diff
@@ -173,7 +218,14 @@ private static function update_grouped_option( $group, $name, $value ) {
 		}
 		$options[ $name ] = $value;
 
-		return update_option( self::$grouped_options[ $group ], $options );
+		if ( self::is_network_option( $name ) ) {
```

Comment:
Will this ever be relevant for grouped options?  Or is it just a case of being thorough and building out future infastructure?